#!/usr/bin/env python
PACKAGE = "to_velocity_tracking"
#import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("k", double_t, 0, "Translation gain", .2, -1.5, 1.5)

gen.add("ktheta", double_t, 0, "Rotation gain", 1.0, -15, 15)

gen.add("power", bool_t, 0, "ON/OFF", True)

gen.add("epsilon", double_t, 0, "Switching threshold", 0.2, -3, 3)

gen.add("u_switch", bool_t, 0, "Switch ux/uy: 1 if true else -1", True)

exit(gen.generate(PACKAGE, "to_velocity_tracking", "ControlVariables"))